lib/network/http/server.js

```diff
@@ -323,6 +351,34 @@ class Server {
     }
 
     /**
+     * Function called when request with an HTTP Expect header is received,
+     * where the value is not 100-continue
+     *
+     * @param {http.IncomingMessage|https.IncomingMessage} req - Request object
+     * @param {http.ServerResponse|https.ServerResponse} res - Response object
+     * @return {undefined}
+     */
+    _onCheckExpectation(req, res) {
+        // Setting no delay of the socket to the value configured
+        req.connection.setNoDelay(this.isNoDelay());
```

Comment:
Why it this done here, any link with `Expect` header? Shouldn't it be done in the top-level request handler instead?